Backendless/Service/UserService.cs

```diff
@@ -472,8 +472,47 @@ private void HandleUserLogin( Dictionary<string, object> invokeResult, bool stay
       }
     }
 
+    public BackendlessUser LoginAsGuest()
+    {
+      return LoginAsGuest(false);
+    }
+
+    public BackendlessUser LoginAsGuest(bool stayLoggedIn)
+    {
+      lock (currentUserLock)
+      {
+        HandleUserLogin(Invoker.InvokeSync<Dictionary<string, object>>(USER_MANAGER_SERVER_ALIAS,

```

Comment:
coding standards - spaces around arguments, space after `,` in the argument list.